screens/LeaderBoardScreen.tsx

```diff
@@ -1,73 +1,73 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Box, ScrollView, Flex, VStack, Text, Image, HStack } from 'native-base';
 import { RootTabScreenProps } from '../types';
 import ButtonWithFocus from '../components/ButtonWithFocus';
 import UsernameWithRank from '../components/UsernameWithRank';
 import styles from '../css/DashboardScreenStyles';
-import FirstPlace from '../assets/images/firstPlace.svg';
-import SecondPlace from '../assets/images/secondPlace.svg';
-import ThirdPlace from '../assets/images/thirdPlace.svg';
+import { useAppDispatch, useAppSelector } from '../stores/hooks';
+import { getDepartmentsRanking, getPersonalRanking } from '../stores/slices/leaderboardSlice';
+import TopRanking from '../components/TopRanking';
 
 export default function LeaderboardScreen({ navigation }: RootTabScreenProps<'Leaderboard'>) {
 
-  const leafGreen = require('../assets/images/leafGreen.png');
+  let {personalRanking, departmentsRank} = useAppSelector((state) => state.leaderboard);
+  let dispatch = useAppDispatch();
+  let [buttonOption, setButtonOption] = useState('Personal');
+
+  let pos = 4;
+
```

Comment:
what is this?

Reply:
it's the initial position for the ranking table (given that the first 3 are special components). I'm using it to add the position number in the table through the map